lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -67,8 +73,9 @@
                                 valueAttribute: 'pref-lang',
                                 required: true,
                                 refreshPageOnSave: true,
-                                helpMessage:
-                                    gettext('The language used for the edX site. The site is currently available in a limited number of languages.'),
+                                helpMessage: interpolate_text(
+                                    gettext('The language used for the {platform_name} site. The site is currently available in a limited number of languages.'), {platform_name: platformName}
```

Comment:
The language used throughout this site. This site is currently available in a limited number of languages.